staging/src/k8s.io/cli-runtime/pkg/printers/tableprinter.go

```diff
@@ -212,18 +212,19 @@ func printTable(table *metav1.Table, output io.Writer, options PrintOptions) err
 				case string:
 					print := val
 					truncated := false
-					// truncate at newlines
-					newline := strings.Index(print, "\n")
-					if newline >= 0 {
+					// Truncate at the first newline, carriage return or formfeed
+					// (treated as a newline by tabwriter).
+					breakchar := strings.IndexAny(print, "\f\n\r")
+					if breakchar >= 0 {
 						truncated = true
-						print = print[:newline]
+						print = print[:breakchar]
 					}
-					fmt.Fprint(output, print)
+					WriteEscaped(output, print)
 					if truncated {
 						fmt.Fprint(output, "...")
 					}
 				default:
-					fmt.Fprint(output, val)
+					WriteEscaped(output, fmt.Sprint(val))
```

Comment:
Sorry, I'm not fully aware of the context of this change. Why we are `fmt.Sprint(val)`?

Reply:
It's to get val into a `string` (i.e. via Stringer or the rules fmt uses), it made sense to me to make the new `WriteEscaped` take a string, rather than put the formatting over there, which might not match all caller's expectations.